pages/_app.tsx

```diff
@@ -287,10 +288,10 @@ export default withRedux(initializeStore)(class MyApp extends App<Props> {
     // If page uses a query parameter to show a modal on page load,
     // then update history so the query parameter version is not last in history.
     const urlParams = new URLSearchParams(window.location.search)
-    const paramLogin = urlParams.get('login')
-    const paramForgotPassword = urlParams.get('forgotPassword')
-    const paramResetPassword = urlParams.get('resetPassword')
-    const paramSendVerificationEmail = urlParams.get('sendVerificationEmail')
+    const paramLogin = urlParams.get(PV.core.login)
+    const paramForgotPassword = urlParams.get(PV.core.forgotPassword)
+    const paramResetPassword = urlParams.get(PV.core.resetPassword)
+    const paramSendVerificationEmail = urlParams.get(PV.core.sendVerificationEmail)
```

Comment:
So these "urlParams" are query params that are passed in the URL.

For example, https://podverse.fm?resetPassword=true

I'm going to move them into the query constants file.